src/renderer/widgets/AddPureProxiedModal/model/form-model.ts

```diff
@@ -41,16 +58,17 @@ const feeChanged = createEvent<string>();
 const isFeeLoadingChanged = createEvent<boolean>();
 const isProxyDepositLoadingChanged = createEvent<boolean>();
 
-const $proxyDeposit = restore(proxyDepositChanged, '0');
-const $isProxyDepositLoading = restore(isProxyDepositLoadingChanged, false);
-const $multisigDeposit = restore(multisigDepositChanged, '0');
-const $fee = restore(feeChanged, '0');
-const $isFeeLoading = restore(isFeeLoadingChanged, false);
+const $fee = createStore<string>('0').reset(formSubmitted);
```

Comment:
If I return from confirm screen to the form screen, seems that I won't see previous data